languages/grammar/fr.json

```diff
@@ -0,0 +1,177 @@
+{
+    "meta": {
+        "regExpFlags": "gi"
+    },
+    "v5": {
+        "article": [
+            ["^ Acc[èe]s ", " l’accès "],
+            ["^ Aire ", " l’aire "],
+            ["^ All[ée]e ", " l’allée "],
+            ["^ Anse ", " l’anse "],
+            ["^ (L['’])?Autoroute ", " l’autoroute "],
+            ["^ Avenue ", " l’avenue "],
+            ["^ Barreau ", " le barreau "],
+            ["^ Boulevard ", " le boulevard "],
+            ["^ Chemin ", " le chemin "],
+            ["^ Petit[\\- ]Chemin ", " le petit chemin "],
+            ["^ Cit[ée] ", " la cité "],
+            ["^ Clos ", " le clos "],
+            ["^ Corniche ", " la corniche "],
+            ["^ Cour ", " la cour "],
+            ["^ Cours ", " le cours "],
+            ["^ D[ée]viation ", " la déviation "],
+            ["^ Entr[ée]e ", " l’entrée "],
+            ["^ Esplanade ", " l’esplanade "],
+            ["^ Galerie ", " la galerie "],
+            ["^ Impasse ", " l’impasse "],
+            ["^ Lotissement ", " le lotissement "],
+            ["^ Mont[ée]e ", " la montée "],
+            ["^ Parc ", " le parc "],
+            ["^ Parvis ", " le parvis "],
+            ["^ Passage ", " le passage "],
+            ["^ Place ", " la place "],
+            ["^ Petit[\\- ]Pont ", " le petit-pont "],
+            ["^ Pont ", " le pont "],
+            ["^ Promenade ", " la promenade "],
+            ["^ Quai ", " le quai "],
+            ["^ Rocade ", " la rocade "],
+            ["^ Rond[\\- ]?Point ", " le rond-point "],
+            ["^ Route ", " la route "],
+            ["^ Rue ", " la rue "],
+            ["^ Grande Rue ", " la grande rue "],
+            ["^ Sente ", " la sente "],
+            ["^ Sentier ", " le sentier "],
+            ["^ Sortie ", " la sortie "],
+            ["^ Souterrain ", " le souterrain "],
+            ["^ Square ", " le square "],
+            ["^ Terrasse ", " la terrasse "],
+            ["^ Traverse ", " la traverse "],
+            ["^ Tunnel ", " le tunnel "],
+            ["^ Viaduc ", " le viaduc "],
+            ["^ Villa(ge)? ", " le villa$1 "],
```

Comment:
What are we looking to match other than Village? Unfortunately `Villa` is feminine in French (la villa)

Reply:
Thank you, I add other rules for `Villa` then